docs/_data/sidebars/_documentation.yml

```diff
@@ -14,6 +14,9 @@ entries:
 
               - title: werf.yaml template engine
                 url: /usage/project_configuration/werf_yaml_template_engine.html
+                
+              - title: Debug werf.yaml
+                url: /usage/project_configuration/debug_werf_yaml.html
```

Comment:
Looking at the content I'm not sure it really deserves a separate page. Maybe in the end of `werf.yaml` templating?